components/rule/index.css

```diff
@@ -12,47 +12,54 @@ governing permissions and limitations under the License.
 
 @import '../commons/index.css';
 
+:root {
+  --spectrum-divider-vertical-height: 100%;
+
+  /* Work around DNA misspelling */
+  --spectrum-divider-small-vertical-width: var(--spectrum-divider-small-vertical-width, var(--spectrum-divider-small-veritcal-width));
+}
+
 .spectrum-Rule {
   width: 100%;
 
   /* Show the overflow for hr in Edge and IE. */
   overflow: visible;
 
   border: none;
-  border-width: var(--spectrum-rule-medium-height);
-  border-radius: var(--spectrum-rule-medium-height);
+  border-width: var(--spectrum-divider-medium-height);
+  border-radius: var(--spectrum-divider-medium-height);
```

Comment:
All tokens renamed from `-rule` to `-divider` since the component was renamed in DNA. The previous tokens are still there, but are missing the new `vertical-width` tokens.